controllers/productController.js

```diff
@@ -0,0 +1,18 @@
+const prodcutService = require('../services/productService');
+const appError = require('../middlewares/appError');
+
+const createReview = async (req, res) => {
+    const { content, userId, } = req.body;
+
+    const { productId } = req.params;
+
+    if (!content || !userId || !productId) throw new appError('KEY_ERROR', 400);
```

Comment:
여기서 userId 부분은 지우는게 좋겠습니다.
userId에 대한부분은 middleware에서 검증이 끝나고, 만약에 user가 존재하지 않는 경우 middlware에서 error를 반환하는 것이 맞습니다.

```suggestion
    if (!content || !productId) throw new appError('KEY_ERROR', 400);
```